interfaces/acados_template/acados_template/custom_update_templates/custom_update_function_zoro_template.in.c

```diff
@@ -692,6 +814,8 @@ static void uncertainty_propagate_and_update(ocp_nlp_solver *solver, ocp_nlp_in
         {%- for it in zoro_description.idx_uh_t %}
         custom_mem->d_uh_tightened[{{it}}] = custom_mem->d_uh[{{it}}]
                         - backoff_scaling_gamma * sqrt(blasfeo_dgeex1(&custom_mem->temp_beta_mat, {{it}}, {{it}}));
+
+        // TODO: make sure uh > lh?
```

Comment:
maybe add to PR description? seems quite relevant :grimacing:

Reply:
This is no new requirement. I removed the comment.
It is of course a problem, if tightened constraints lead to an empty feasible set.
In this case, the next OCP solver call should return an infeasibility status, which I guess is the expected behavior.